packages/components/socialFeed/NewsFeed/MapModal/index.tsx

```diff
@@ -0,0 +1,295 @@
+import React, { Suspense, lazy, useEffect, useState } from "react";
```

Comment:
This component is too long, try to split it on your own 👍

Reply:
done